### ISSUE TITLE
v0.11.0 in Windows has error 

### ISSUE THREAD

**Issue by agrozyme** [davidmyersdev/vite-plugin-node-polyfills#29]
v0.11.0 has Error, But v0.9.0 is OK
```
[plugin:vite:import-analysis] Failed to resolve import "D:EnterpriseTaiheite-quasar
ode_modulesite-plugin-node-polyfillsshimsdistindex.js" from "src\main.ts". Does the file exist?
D:/Enterprise/Taihe/vite-quasar/src/main.ts:1:43
1  |  import { Buffer as BufferPolyfill } from 'D:\Enterprise\Taihe\vite-quasar\node_modules\vite-plugin-node-polyfills\shims\dist\index.js'
   |                                            ^
2  |  window.Buffer = BufferPolyfill
3  |  import { global as globalPolyfill } from 'D:\Enterprise\Taihe\vite-quasar\node_modules\
```
![Screenshot of Vite + Vue + TS](https://github.com/davidmyersdev/vite-plugin-node-polyfills/assets/13079345/8b3257c5-84f3-4c84-ab70-ef4448b8a0ea)

**Comment by 6abotage** [comment]
Hello, I just tested v0.11.0 on my mac and I also have an error with the buffer polyfill. It works in v0.9.0 and v0.10.0
So I don't think the problem is based on the operating system windows

**Comment by vexxvakan** [comment]
Can confirm here as well. 0.10.0 is working fine, 0.11.0 does not at all.

- Mac
- Vite 4.4.9

**Comment by davidmyersdev** [comment]
Fixed in `v0.11.1`.
